.github/workflows/smart-contracts.yml

```diff
@@ -2,7 +2,7 @@ name: Smart Contracts
 
 on:
   pull_request:
-    paths: ['smart-contracts/**']
+    paths: ['.github/**,smart-contracts/**']
```

Comment:
why the `.github/**` ?

Reply:
That was from me, and I added it after I made a mistake in the yml file itself:

```
commit d144d9e1501a91648ef16452e864d1f40f1a9518 (feature/code-cov-contracts)
Author: James Moore <james@sifchain.finance>
Date:   Tue Dec 29 15:27:54 2020 -0800

    Run github actions when you change the actions file themselves

diff --git a/.github/workflows/smart-contracts.yml b/.github/workflows/smart-contracts.yml
index 50506a98..df3417ca 100644
--- a/.github/workflows/smart-contracts.yml
+++ b/.github/workflows/smart-contracts.yml
@@ -2,7 +2,7 @@ name: Smart Contracts
 
 on:
   pull_request:
-    paths: ['smart-contracts/**']
+    paths: ['.github/**,smart-contracts/**']
 
 jobs:
   test:

commit 47cf308adc76183c73024140d29bc107972bf520
Author: James Moore <james@sifchain.finance>
Date:   Tue Dec 29 15:26:21 2020 -0800

    Fixed dash

diff --git a/.github/workflows/smart-contracts.yml b/.github/workflows/smart-contracts.yml
index 5d720030..50506a98 100644
--- a/.github/workflows/smart-contracts.yml
+++ b/.github/workflows/smart-contracts.yml
@@ -32,6 +32,4 @@ jobs:
         run: npm run test
 
       - name: Test Code Coverage
-      - run: npm run coverage
-
-
+        run: npm run coverage


```

Reply:
In order to try to let changes in github actions to run stuff.  Backing it out for now as syntax has problems.

Reply:
oh I get it. Thanks